custom/hardened-alpine/infra/Dockerfile

```diff
@@ -37,7 +37,10 @@ RUN apk add --no-cache ca-certificates
 RUN adduser -s /bin/true -u 1000 -D -h $APP_DIR $APP_USER \
   && mkdir "$DATA_DIR" "$CONF_DIR" \
   && chown -R "$APP_USER" "$APP_DIR" "$CONF_DIR" \
-  && chmod 700 "$APP_DIR" "$DATA_DIR" "$CONF_DIR"
+  && chmod 700 "$APP_DIR" "$DATA_DIR" "$CONF_DIR" \
```

Comment:
Thanks @tuananh, Can we also give the APP_USER=1001 (in place of `litmus`) [here](https://github.com/litmuschaos/test-tools/blob/6dab915e54c01f4ff4a6b192ee618b40400affaf/custom/hardened-alpine/infra/Dockerfile#L15) ? So that we don't need to add `anyuser` SCC to the sa (clusterrole).